broker/configuration.go

```diff
@@ -63,20 +64,32 @@ func SaramaConfigFromBrokerConfig(cfg Configuration) (*sarama.Config, error) {
 	if strings.Contains(cfg.SecurityProtocol, "SSL") {
 		saramaConfig.Net.TLS.Enable = true
 	}
-	if cfg.CertPath != "" {
+
+	if strings.EqualFold(cfg.SecurityProtocol, "SSL") {
 		tlsConfig, err := tlsutils.NewTLSConfig(cfg.CertPath)
```

Comment:
Just want to confirm ... since apps will soon need to support running without a custom CA cert -- does this work when cfg.CertPath is `""`?

Reply:
Nope, and it was failing in our unit tests. Fixed in the new version. Thank you for noting @bsquizz !